### DISCUSSION TITLE
Using OneDrive to synchronize and produce different file paths

### DISCUSSION THREAD

**Discussion by talisman-B** [remotely-save/remotely-save#524] [Q&A] [answered]
Hello, I am trying to use remotely-save and OneDrive to synchronize window and iPad, but I have encountered some issues that cannot be resolved. According to the instructions, I have completed authentication and ensured that the database names on different platforms are the same. I can connect to OneDrive, but I can't sync.
I tried to proactively refresh and encountered the issue on the image.I don't know why OneDrive generates OneDrive \ app \ Graph \ My Library instead of OneDrive \ app \ remotely-save \ My Library .I'm not sure if the problem is here, but I really need your help.
![1](https://github.com/remotely-save/remotely-save/assets/81567105/707cf724-ea4a-41c6-8e98-a8ed7ee4d93c)

**Comment by fyears** [comment, marked as answer]
你是个人版 onedrive 吗？有些用户反馈这个问题但是我自己无法复现，不知道咋整

**Reply by fyears** [reply]
dup of #517

**Comment by fyears** [comment]
dup of #517
